src/main/java/com/jaspersoft/jasperserver/jaxrs/client/dto/settings/AwsSettings.java

```diff
@@ -0,0 +1,133 @@
+        return productTypeIsJrsAmi;
+    }
+
+    public AwsSettings setProductTypeIsJrsAmi(Boolean productTypeIsJrsAmi) {
+        this.productTypeIsJrsAmi = productTypeIsJrsAmi;
+        return  this;
+    }
+
+    public List<String> getAwsRegions() {
+        return awsRegions;
+    }
+
+    public AwsSettings setAwsRegions(List<String> awsRegions) {
+        this.awsRegions = awsRegions;
+        return  this;
+    }
+
+    public Boolean isProductTypeIsMpAmi() {
+        return productTypeIsMpAmi;
+    }
+
+    public AwsSettings setProductTypeIsMpAmi(Boolean productTypeIsMpAmi) {
+        this.productTypeIsMpAmi = productTypeIsMpAmi;
+        return  this;
+    }
+
+    public Boolean isSuppressEc2CredentialsWarnings() {
+        return suppressEc2CredentialsWarnings;
+    }
+
+    public AwsSettings setSuppressEc2CredentialsWarnings(Boolean suppressEc2CredentialsWarnings) {
+        this.suppressEc2CredentialsWarnings = suppressEc2CredentialsWarnings;
+        return  this;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof AwsSettings)) return false;
+
+        AwsSettings that = (AwsSettings) o;
+
+        if (productTypeIsEc2 != null ? !productTypeIsEc2.equals(that.productTypeIsEc2) : that.productTypeIsEc2 != null)
+            return false;
+        if (isEc2Instance != null ? !isEc2Instance.equals(that.isEc2Instance) : that.isEc2Instance != null)
+            return false;
+        if (productTypeIsJrsAmi != null ? !productTypeIsJrsAmi.equals(that.productTypeIsJrsAmi) : that.productTypeIsJrsAmi != null)
+            return false;
+        if (getAwsRegions() != null ? !getAwsRegions().equals(that.getAwsRegions()) : that.getAwsRegions() != null)
+            return false;
+        if (productTypeIsMpAmi != null ? !productTypeIsMpAmi.equals(that.productTypeIsMpAmi) : that.productTypeIsMpAmi != null)
+            return false;
+        return !(suppressEc2CredentialsWarnings != null ? !suppressEc2CredentialsWarnings.equals(that.suppressEc2CredentialsWarnings) : that.suppressEc2CredentialsWarnings != null);
+
+    }
+
+    @Override
+    public int hashCode() {
+        int result = productTypeIsEc2 != null ? productTypeIsEc2.hashCode() : 0;
+        result = 31 * result + (isEc2Instance != null ? isEc2Instance.hashCode() : 0);
+        result = 31 * result + (productTypeIsJrsAmi != null ? productTypeIsJrsAmi.hashCode() : 0);
+        result = 31 * result + (getAwsRegions() != null ? getAwsRegions().hashCode() : 0);
+        result = 31 * result + (productTypeIsMpAmi != null ? productTypeIsMpAmi.hashCode() : 0);
+        result = 31 * result + (suppressEc2CredentialsWarnings != null ? suppressEc2CredentialsWarnings.hashCode() : 0);
+        return result;
+    }
+
+    @Override
+    public String toString() {
+        String awsRegions = "";
+        for (String awsRegiono : this.awsRegions) {
+            awsRegions += awsRegions + ", ";
```

Comment:
this.awsRegions.toString() will do it for you, but please check awsRegions for null first